package-lock.json

```diff
@@ -90,7 +90,7 @@
         "react-native-linear-gradient": "^2.8.1",
         "react-native-localize": "^2.2.6",
         "react-native-modal": "^13.0.0",
-        "react-native-onyx": "1.0.89",
+        "react-native-onyx": "1.0.93",
```

Comment:
@ospfranco diff here is wrong. Could you rectify this?

Reply:
I don't understand why is this showing .93. You can see on the changed files it is .94

https://github.com/Expensify/App/pull/28299/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R136